sdk/python/kubeflow/training/api/training_client.py

```diff
@@ -355,6 +356,9 @@ def create_job(
                 set, Base Image must support `bash` CLI to execute the training script.
             parameters: Dict of input parameters that training function might receive.
             num_workers: Number of Worker replicas for the Job.
+            num_procs_per_worker: Number of processes per PyTorchJob worker for `torchrun` CLI. You
```

Comment:
It seems that this option is only for PyTorchJob, but this API is not only for PyTorchJob.
Do you aim to support other frameworks as well?
If not, pytorch prefix or suffix would be better.

Reply:
@tenzen-y I think, we support TF and PyTorch today when this parameter is used: https://github.com/kubeflow/training-operator/blob/master/sdk/python/kubeflow/training/api/training_client.py#L445-L460.
I added validation to have correct setting for TFJob and PyTorchJob.

Reply:
That makes sense.
Thanks.
/lgtm
/approve